.coveragerc

```diff
@@ -24,7 +24,7 @@ exclude_lines =
     if __name__ == .__main__.:
 
     # Don't complain about abstract methods, they aren't run:
-    @(abc\.)?abstractmethod
+    @(abc\.)?abstract(((class|static)?method)|property)
```

Comment:
so tuning to get better score than me ? ;-)

Reply:
I already have 99% on this one man... it's another league :rofl: